ci/common

```diff
@@ -1,100 +1,6 @@
 #!/bin/sh
 # Use https://www.shellcheck.net/ to reduce mistakes if you make changes to this file.
 
-remove_docker_mirror_on_travis() {
-    # This is a workaround to an issue caused by the existence of a docker
-    # registry mirror in our CI environment. Without this fix that removes the
-    # mirror, chartpress fails to realize the existence of already built images
-    # and rebuilds them.
-    #
-    # ref: https://github.com/moby/moby/issues/39120
-    sudo cat /etc/docker/daemon.json
-    echo '{"mtu": 1460}' | sudo dd of=/etc/docker/daemon.json
-    sudo systemctl restart docker
-    docker ps -a
-}
-
-setup_helm () {
-    echo "setup helm ${HELM_VERSION}"
-    curl -sf https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | DESIRED_VERSION=${HELM_VERSION} bash
-}
-
-setup_k3s () {
-    echo "setup k3s ${K3S_VERSION}"
-    # https://rancher.com/docs/k3s/latest/en/installation/install-options/how-to-flags/
-
-    # NOTE: k3s has a Network Policy controller called kube-router, but it is
-    #       not robust enough for use, so we disable it and install our own:
-    #       calico. --flannel-backend=none should not be passed if we don't want
-    #       to install our own CNI.
-    #
-    #       ref: https://github.com/rancher/k3s/issues/947#issuecomment-627641541
-    #
-    # NOTE: k3s 1.16 and older needed a flag named --no-deploy instead of
-    #       --disable.
-    #
-    curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=${K3S_VERSION} sh -s - \
-        --write-kubeconfig-mode=644 \
-        "${k3s_disable_command:---disable}" metrics-server \
-        "${k3s_disable_command:---disable}" traefik \
-        "${k3s_disable_command:---disable}" local-storage \
-        --disable-network-policy \
-        --flannel-backend=none \
-        --docker \
- || {
-        echo "Setup of k3s failed!"
-        systemctl status k3s.service
-        journalctl --no-pager --utc | tail -n 100
-        exit 1
-    }
-    export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
-
-    setup_calico
-}
-
-setup_calico() {
-    # Install calico as a CNI to enforce our NetworkPolicies. Note that canal
-    # could do this job as well, but we failed to set it up in Travis CI.
-    #
-    # Below we download the calico.yaml Kubernetes manifest and insert a
-    # container_settings section just below the phrase '"type": "calico"' and
-    # then `kubectl apply` it.
-    #
-    # ref: https://rancher.com/docs/k3s/latest/en/installation/network-options/
-    #
-    curl -sfL https://docs.projectcalico.org/v3.14/manifests/calico.yaml \
-  | sed '/"type": "calico"/a\
-          "container_settings": {\
-            "allow_ip_forwarding": true\
-          },' \
-  | kubectl apply -f -
-}
```

Comment:
setup_calico is referenced in the CONTRIBUTING.md file. I suggest to copy paste the content of it there or retain it here.

await_... functions are also referenced in CONTRIBUTING.md in comments, which can simply be cut away.

Can you give the CONTRIBUTING.md file an update with regards to this?

Reply:
Will do!

Reply:
I pushed a commit to resolve this!